docs/migration-v8.md

```diff
@@ -1 +1,515 @@
 # Migration guide from v7 to v8
+
+Welcome to the migration guide for Polly's v8 version. The v8 version of Polly brings major new enhancements and supports all the same scenarios as previous versions. In the following sections, we'll detail the differences between the v7 and v8 APIs and provide steps on how to transition smoothly.
+
+> [!NOTE]
+> The v7 API is still available and fully supported even when using the v8 version.
+
+## Major differences
+
+- **The term *Policy* is now replaced with *Strategy***: In previous versions, Polly used the term *policy* for retries, timeouts, etc. In v8, these are referred to as *resilience strategies*.
+- **Introduction of Resilience Pipelines**: The [resilience pipeline](pipelines/index.md) combines one or more resilience strategies. This is the foundational API for Polly v8, similar to the **Policy Wrap** in previous versions but integrated into the core API.
+- **Unification of interfaces**: Interfaces such as `IAsyncPolicy`, `IAsyncPolicy<T>`, `ISyncPolicy`, `ISyncPolicy<T>`, and `IPolicy` are now unified under `ResiliencePipeline` and `ResiliencePipeline<T>`. The resilience pipeline supports both synchronous and asynchronous executions.
+- **Native async support**: Polly v8 was designed with asynchronous support from the start.
```

Comment:
I would probably change this into "Unified sync/async flow" or something like that, because Polly had native async support even in v7, although you had to implement it separately from sync.